src/clients/cli.rs

```diff
@@ -78,8 +86,15 @@ impl Cli {
             command.arg("-e").arg(format!("{}={}", key, value));
         }
 
-        for (orig, dest) in image.volumes() {
-            command.arg("-v").arg(format!("{}:{}", orig, dest));
+        if Self::running_inside_docker()
+            && image.volumes().into_iter().collect::<Vec<_>>().len() > 0
+        {
+            // TODO: Panic here or just log and continue?
+            panic!("Running inside Docker: Unable to map volumes.");
```

Comment:
How should this be handled?

Reply:
This is very restrictive. It is still possible to map volumes if they mapped correctly